### PR TITLE
feat(DEX-3744): Apply Pendo to MFE

### PR DESCRIPTION
**Pull request by keoniWGU**
# Overview
Add Pento feature flag and the snippet env variable on index.jsx.

## Checklist

⚠️ **Please make sure to fill this checklist before asking for reviews.**

- [x] Code is correctly formatted and linted
- [x] Unit tests are updated and are passing
- [x] PR title follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary):
      i.e. `feat(CUR-###): Feature title`
- [x] PR Title aligns to Semantic-Release [supported prefixes](https://www.conventionalcommits.org/en/v1.0.0/#examples) (NOTE: prefixes are case sensitive, keep lower-case):

```diff
 feat() - Feature (0.X.0)
 fix() - Patch (0.0.X)
 docs() - Patch (0.0.X), will only scope to README changes
 refactor() - Patch (0.0.X)
 revert() - Patch (0.0.X)
 style() - Patch (0.0.X)
```

- [x] Check for unused files
- [x] Check work before asking for reviewers
- [x] Fix any linting errors
- [x] SECURITY: No secrets where commited to the repo
- [x] COMPLIANCE: Commited code is not propietary and adheres to Open Source licensing

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -14,11 +14,23 @@ import initializeStore from './app/store';
 import CourseView from './features/course-view/CourseView';
 import Layout from './features/layout/Layout';
 
+ensureConfig([
+  'DISABLE_APP_HEADER',
+], 'CoursePlayer component');
+
+ensureConfig([
```
Comment (rodmgwgu) [inline src/index.jsx:21]: This can be merged with the previous ensureConfig call, just add your name to the array

Comment (rodmgwgu) [reply src/index.jsx:21]: The seconf string is for identifying where the issue is if the variable was missing, you will get a log with something like "ENABLE_PENDO config missing in CoursePlayer component"

**Review by rodmgwgu — CHANGES_REQUESTED** [review]
You need to also add your variables to the main index.jsx file, in the "mergeConfig" call, and make sure to cast the boolean there

File: .env [hunk 2]
```diff
@@ -11,6 +11,7 @@ DISCUSSIONS_MFE_BASE_URL=''
 ECOMMERCE_BASE_URL=''
 ENABLE_JUMPNAV='true'
 ENABLE_NOTICES=''
+ENABLE_PENDO =true
```
Comment (rodmgwgu) [inline .env:14]: Please remove extra space before "="

File: src/index.jsx [hunk 3]
```diff
@@ -55,6 +67,7 @@ initialize({
         ENTERPRISE_LEARNER_PORTAL_HOSTNAME: process.env.ENTERPRISE_LEARNER_PORTAL_HOSTNAME || null,
         ENABLE_JUMPNAV: process.env.ENABLE_JUMPNAV || null,
         ENABLE_NOTICES: process.env.ENABLE_NOTICES || null,
+        ENABLE_PENDO: process.env.ENABLE_PENDO === 'true' || null,
```
Comment (rodmgwgu) [inline src/index.jsx:70]: You also need to add the PENDO_SNIPPET var here

**Review by AngelRicardezWGU — APPROVED** [review]

**Pull request merged**